2016/macros.ddoc

```diff
@@ -79,7 +79,7 @@ $(DIV class="page_right equal",
   )
   $(DIV class="twitter",
     $(DIV class="hash", )
-<a class="twitter-timeline" href="https://twitter.com/search?q=%23dconf+%23dlang" data-widget-id="405059685734023169">Tweets about "#dconf #dlang"</a>
+<a class="twitter-timeline" href="https://twitter.com/search?q=%23dconf+%23dlang-RT" data-widget-id="405059685734023169">Tweets about "#dconf #dlang"</a>
```

Comment:
I'm afraid it's not that simple. `-RT` has to be added in the settings at Twitter. Either the owner of the current widget (@andralex?) adds it, or you make a new one from your account and change the `id` here.

I'd prefer it if the current one would be changed. I have control over the one on dlang.org, but I don't think I should.